src/main/java/ArrEx6.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrEx6 {
+    public String findMaxLenght(int[] arr) {
+        List<String> result = new ArrayList<>();
+        int n = arr.length;
+        int length = 1;
+        int max = 1;
+        int start = 0;
+        int end = 0;
+        for (int i = 0; i < n - 1; i++) {
+            if (arr[i + 1] - arr[i] == 1) {
+                length++;
+                if (length > max) {
+                    max = length;
+                    start = i + 2 - length;
+                    end = i + 2;
+                }
+            } else {
+                length = 1;
+            }
+        }
+        for (int i = start; i < end; i++)
```

Comment:
Đưa lệnh khai báo xuống ngay đây
```suggestion
        List<String> result = new ArrayList<>();
        for (int i = start; i < end; i++)
```

Reply:
Done